dali/kernels/signal/dct/dct_test.h

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
```

Comment:
If it's exactly the same code as in table.h I'd use that directly instead

Reply:
It's not exactly the same code. The reference implementation from here computes the whole transform on a one row of input. It's simpler than the proper end to end implementation, but it does more than just calculating cosine table (and that is done in table.h). It's convenient for tests.